Sources/Bootstrap/Tags/ButtonGroup.swift

```diff
@@ -0,0 +1,72 @@
+import Leaf
+import TemplateKit
+
+/// Bootstrap Button Group Tag
+public final class ButtonGroupTag: TagRenderer {
+    enum GroupKeys: String {
+        case standard = "btn-group"
+        case vertical = "btn-group-vertical"
+    }
+
+    private static let paramCount: Int = 3
+
+    public func render(tag: TagContext) throws -> EventLoopFuture<TemplateData> {
+
+        let body = try tag.requireBody()
+
+        guard tag.parameters.count <= ButtonGroupTag.paramCount else {
+            throw tag.error(reason: "Wrong Parameter Count: \(tag.parameters.count) Expecting <= \(ButtonGroupTag.paramCount)")
```

Comment:
There's a requireParameterCount function for this right?

Reply:
There is... but is a == on the count...
seems like a opportunity to make a extension there

Reply:
ah, I see! yeah then this is necessary. A PR to `TemplateKit` could be nice if you're feeling up for it.